helm/aws-load-balancer-controller/values.yaml

```diff
@@ -104,6 +104,9 @@ additionalLabels: {}
 # Enable cert-manager
 enableCertManager: false
 
+# Cluster DNS domain (required for generating the TLS certificates)
+clusterDomain: cluster.local
```

Comment:
Lets group it together with clusterName in the values.yaml file.
Also, lets have it as follows:
```
# cluster contains configurations specific to the kubernetes cluster
cluster:
    domain: cluster.local
```
In future, we can nest the clusterName inside as well.

Reply:
Thanks for your feedback, I have refactored it to `cluster.dnsDomain`.